### PR TITLE
[Libtask] make libtask don't depend on the memory layout of jl_task_t

### PR DESCRIPTION
**Pull request by KDr2**

### CODE REVIEW

File: L/Libtask/build_tarballs.jl [hunk 1]
```diff
@@ -5,7 +5,7 @@ using BinaryBuilder, Pkg
 julia_version = v"1.6.0"
```
Comment (giordano) [inline L/Libtask/build_tarballs.jl:5]: Just to be kinda safer, maybe you should build against libjulia 1.3?

**Review by giordano — COMMENTED** [review]
Related: have you seen https://github.com/TuringLang/Libtask.jl/issues/85?

File: L/Libtask/build_tarballs.jl (hunk 1 above)
Comment (KDr2) [reply L/Libtask/build_tarballs.jl]: Update.

**Comment by KDr2** [comment]
> Related: have you seen [TuringLang/Libtask.jl#85](https://github.com/TuringLang/Libtask.jl/issues/85)?

Thanks, I've commented on that issue.

**Pull request merged**